modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -75,10 +86,29 @@ object GlobalL0Service {
         }
 
       def pullGlobalSnapshots: F[Either[(Hashed[GlobalIncrementalSnapshot], GlobalSnapshotInfo), List[Hashed[GlobalIncrementalSnapshot]]]] =
-        lastGlobalSnapshotStorage.getOrdinal.flatMap {
+        lastGlobalSnapshotStorage.get.flatMap {
           _.fold {
             pullLatestSnapshot.map(_.asLeft[List[Hashed[GlobalIncrementalSnapshot]]])
-          } { lastStoredOrdinal =>
+          } { lastSnapshot =>
+            val lastStoredOrdinal = lastSnapshot.ordinal
+
+            def realign(priorityPeers: NonEmptySet[PeerId]): F[Unit] =
+              for {
+                peers <- priorityPeers.toNonEmptyList
```

Comment:
shouldn't you use all the peers but priority peers with bigger probability of selection?
the way it's implemented right now it doesn't mean they are priority but they are "source of truth" peers which means the network is much more centralized

cc: @ryle-goehausen

Reply:
Summary of discussion on this topic and proposed plan:

@sdavidp we should add a flag here to distinguish between these two cases. Default should be to rely on priority peers, but if the flag is present or enabled (through CLI) we should revert to using a majority function on the seed list.

Again reminder this code should only be invoked on the currency L0 / L1, and only for the usage of the bridge to GL0. We have ~4 remaining issues to solve before we can drop the requirement on using priority peers entirely. But we should expose the flag to use the majority for any users that are capable of setting up all the ops scripts to handle rollbacks manually. For users who do not have enough devops setup to handle all the edge cases manually (rollbacks etc,) then we'll have to rely on some centralization for bridge between GL0 and CL0. 

Just to re-iterate: This PR should NOT affect majority functions for CL0 internally, nor majority functions for GL0 internally, but only on the interaction point between them

Reply:
Resolving; code no longer present